test/util.js

```diff
@@ -75,7 +75,14 @@ exports.setup = function setup () {
     fs.mkdirp(WORK_CWD, function (err) {
       if (err) t.end(err)
       process.chdir(WORK_CWD)
-      t.end()
+
+      if (process.platform === 'darwin' && process.env.TRAVIS) {
+        fs.remove(path.join(process.env.HOME, '.wine'), function () {
```

Comment:
Yeah, this was just an attempt to see what could be causing it to fail on CI, will revert after investigating more.